Description: Pedantic complaits by lintian.
 Spelling in ftp/ftp.1 and ftp/cmds.c.
 .
 Incorrect use of hyphenation in ftp/ftp.1.
Author: Mats Erik Andersson <debian@gisladisker.se>
Forwarded: no
Last-Update: 2010-02-27
--- netkit-ftp-0.17/ftp/ftp.1.debian
+++ netkit-ftp-0.17/ftp/ftp.1
@@ -343,7 +343,7 @@
 seconds.
 If
 .Ar seconds
-is ommitted, the current inactivity timer is printed.
+is omitted, the current inactivity timer is printed.
 .It Ic lcd Op Ar directory
 Change the working directory on the local machine.
 If
@@ -854,7 +854,7 @@
 .Ar newmask  .
 If
 .Ar newmask
-is ommitted, the current umask is printed.
+is omitted, the current umask is printed.
 .It Xo
 .Ic user Ar user-name
 .Op Ar password
@@ -957,7 +957,7 @@
 (stdin).
 If the shell command includes spaces, the argument
 must be quoted; e.g.
-\*(Lq" ls -lt"\*(Rq.
+\*(Lq" ls \-lt"\*(Rq.
 A particularly
 useful example of this mechanism is: \*(Lqdir more\*(Rq.
 .It
@@ -1010,9 +1010,9 @@
 .Ic type
 may be one of \*(Lqascii\*(Rq, \*(Lqimage\*(Rq (binary),
 \*(Lqebcdic\*(Rq, and \*(Lqlocal byte size\*(Rq (for
-.Tn PDP Ns -10's
+.Tn PDP Ns \-10's
 and
-.Tn PDP Ns -20's
+.Tn PDP Ns \-20's
 mostly).
 .Nm Ftp
 supports the ascii and image types of file transfer,
--- netkit-ftp-0.17/ftp/cmds.c.debian
+++ netkit-ftp-0.17/ftp/cmds.c
@@ -2511,7 +2511,7 @@
 		if (overbose && 
 		    !strncmp(reply_string, "215 TOPS20", 10))
 			printf(
-"Remember to set tenex mode when transfering binary files from this machine.\n");
+"Remember to set tenex mode when transferring binary files from this machine.\n");
 	}
 	verbose = overbose;
 #else
